clang/test/Driver/Xclangas.s

```diff
@@ -0,0 +1,6 @@
+/// Check that -Xclangas/-Xclangas= are passed to -cc1as.
+// RUN: %clang -### -Xclangas -target-feature -Xclangas=+v5t %s 2>&1 | \
+// RUN:   FileCheck -check-prefix=ARGS %s
+// ARGS: -cc1as
+// ARGS: -target-feature
```

Comment:
Use `-SAME:` whenever applicable

Test `"-target-feature" "+v5"` (ensure that they are adjacent)